.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true
+}
```

Comment:
It seems, we didn't copy this from the old gitlab project. Copying the old settings.json would be more appropriate than beginning a new one, i guess.

Reply:
Copied the one from GitLab to GitHub repo.